app/Listeners/Hold/UnassignHoldOnDisconnect.php

```diff
@@ -5,13 +5,20 @@
 
 use App\Events\HoldUnassignedEvent;
 use App\Events\NetworkAircraftDisconnectedEvent;
+use App\Models\Hold\AssignedHold;
 use App\Models\Vatsim\NetworkAircraft;
 
 class UnassignHoldOnDisconnect
 {
     public function handle(NetworkAircraftDisconnectedEvent $event) : bool
     {
-        event(new HoldUnassignedEvent($event->getAircraft()));
+        $assignedHold = AssignedHold::find($event->getAircraft()->callsign);
```

Comment:
Could find or fail here and exit out of the listener that way? not fussed either way really

Reply:
I believe the exception findOrFail would cause the rest of the event-chain to break